Jenkinsfile

```diff
@@ -1,4 +1,6 @@
-// All non-runtime dependencies installed in Dockerfile.ci
+#!groovy
+@Library('ci-scripts') _
+
 pipeline {
   agent {
```

Comment:
Just use this line, and you won't need `Dockerfile.ci` file

```suggestion
  agent any
```

Reply:
changed